src/Components/BreathingRoom/BreathingRoom.js

```diff
@@ -1,11 +1,41 @@
 import React from 'react';
+import {Component} from 'react'
 import './BreathingRoom.scss';
 
-const BreathingRoom = () => {
+import {inhaleExhale} from './BreathingRoomAnimation.js'
 
-  return (
-    <h1>Breathing Room</h1>
+class BreathingRoom extends Component {
+  constructor(props){
+    super(props)
+    this.state ={
+      isBreathing: false
+    }
+  }
+  clickHandler = () => {
+    this.setState({isBreathing: true})
```

Comment:
Nice use of state to get the directions to display only when needed

Reply:
I agree! The mental transition from using `.querySelectors()` to manipulate the `DOM` to using `state` in React has been an adjustment, for sure. This is a really great way of addressing that!